db/fixtures/miq_product_features.yml

```diff
@@ -2815,6 +2815,10 @@
   :feature_type: node
   :identifier: my_settings
   :children:
+  - :name: View
+    :description: View My Settings
+    :feature_type: view
+    :identifier: my_settings_view
```

Comment:
Note to self - we might want a database migration adding this to any existing role that has one of the other my_settings permissions.